make.cmd

```diff
@@ -75,9 +78,9 @@ if not "%1" == "install" ( goto end )
 
 if not exist %NATIVE_INSTALL%\%ARCH_PATH% ( mkdir %NATIVE_INSTALL%\%ARCH_PATH% )
 @echo on
-copy %LIBGIT2_TARGET%\%CMAKE_CONFIG%\git2.dll %NATIVE_INSTALL%\%ARCH_PATH%
-copy %LIBJAGGED_TARGET%\%CMAKE_CONFIG%\jagged.dll %NATIVE_INSTALL%\%ARCH_PATH%
-copy %LIBJAGGED_TEST_TARGET%\%CMAKE_CONFIG%\jagged_test.dll %NATIVE_INSTALL%\%ARCH_PATH%
+copy %LIBGIT2_TARGET%\libgit2.dll %NATIVE_INSTALL%\%ARCH_PATH%
+copy %LIBJAGGED_TARGET%\libjagged.dll %NATIVE_INSTALL%\%ARCH_PATH%
+copy %LIBJAGGED_TEST_TARGET%\libjagged_test.dll %NATIVE_INSTALL%\%ARCH_PATH%
```

Comment:
I'm not sure why you're doing this.  DLLs on Windows are generally named `foo.dll` (not `libfoo.dll`).  Is this something that MinGW does?  If so, that seems very strange, but in any case - the rest of Windows environments *do not* and this will break all of them.  Please don't assume that the only people running this script are in MinGW.